pkg/artifactory/resource_artifactory_user.go

```diff
@@ -88,10 +91,9 @@ func resourceArtifactoryUser() *schema.Resource {
 			"password": {
 				Type:             schema.TypeString,
 				Sensitive:        true,
-				Required:         true,
-				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsNotEmpty),
-				Description: "Password for the user. Password validation is not done by the provider and is " +
-					"offloaded onto the Artifactory.",
+				Optional:         true,
+				ValidateDiagFunc: validation.ToDiagFunc(defaultPassValidation),
```

Comment:
Oh wait, did we decide *not* to validate? I forgot.

Reply:
Nope. Get out of the password business and fail late